src/src/PluginStructs/P093_data_struct.cpp

```diff
@@ -80,9 +80,50 @@ bool P093_data_struct::read(String& result) const {
   result += F(",\"temperature\":");
   result += toString(_currentValues.temperature, 1) + '}';
 
+  return true;
+}
+
+bool P093_data_struct::plugin_get_config_value(struct EventStruct *event,
+                                               String            & string) {
+  bool success         = true;
```

Comment:
Maybe we could add 
```
if (_valuesInitialized == false) {
    return false;
  }
```
here too to make sure values are/were initialised before being returned?